tools/install.py

```diff
@@ -92,3 +93,9 @@ def run(ip_addr, hostname, config, scripts, user, toolchain, **kwargs):
 
     if config in ['ignore', 'i']:
         cprint('Ignoring configuration changes', 'blue', attrs=['bold'])
+        
+    # Pipe the git commit to file
+    versionFile = open("version.txt", "w")
+    subprocess.Popen("git log -1", stdout=versionFile, shell=True)
```

Comment:
Also you can just do this with subprocess call you don't need popen. In fact that might be dangerous as it's asynchronous.